client-portal/modules/card/components/Detail.tsx

```
@@ -81,25 +81,25 @@
                       : '/static/avatar-colored.svg'
                   )}
-                  alt="profile"
+                  alt='profile'
                 />
                 <div>
                   <CommentContent>
                     <h5>{renderUserFullName(createdUser)}</h5>
                     <div
-                      className="comment"
+                      className='comment'
                       dangerouslySetInnerHTML={{ __html: comment.content }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)